### DISCUSSION TITLE
Upcoming training for the New England region at MIT: MOOSE (Jan 16-17) & BISON (Jan 18-19)

### DISCUSSION THREAD

**Discussion by GiudGiud** [idaholab/moose#26197] [News]
The Idaho National Laboratory is providing a MOOSE training followed by a BISON training this January at MIT in Cambridge, MA. 
 
For the MOOSE workshop on January 16-17:
MOOSE is an open-source finite element & finite volume framework developed by the DOE national laboratories that is the base for much of the advanced nuclear reactor modeling software. There are also several ongoing projects to develop MOOSE applications for fusion device modeling.
 
The Multiphysics Object Oriented Simulation Environment (MOOSE) Workshop provides an in-depth introduction to the framework. The course is two days in length and covers material starting from the mathematical underpinnings of the framework through advanced features of the MOOSE Multiapps system. The course uses a realistic multiphysics problem that is solved with increasing realism and complexity utilizing the many pluggable systems in the framework. Hands on exercises are supplied so attendees are encouraged to bring their own hardware to participate. More information about MOOSE can be found on our [website](https://mooseframework.inl.gov/index.html).
 
The registration deadline will be flexible but we encourage early registration as space is limited. Please email me for any issue or question with registration.
 
 
For BISON workshop on January 18-19th:
The BISON fuel performance team is hosting a two-day, [interactive workshop](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmooseframework.inl.gov%2Fbison%2Ftraining%2Findex.html&data=05%7C01%7CGuillaume.Giudicelli%40inl.gov%7Caedbf4d44d3b4aa8de8908dbd5844fe1%7C4cf464b7869a42368da2a98566485554%7C0%7C0%7C638338537012089739%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=f12GTteBKjWTIicpgUh647ujZt5BzzecQsXq2zaSHyQ%3D&reserved=0). This workshop is intended to help prospective users get started with the code, provide an overview of the methods to model specific fuel types, and provide practical examples to help users gain experience. Please forward this message to anyone at your institution interested in BISON, fuel performance modeling, or MOOSE-related software.
WORKSHOP TOPICS 
•	Introduction to thermomechanical fuel performance modeling and multiphysics analysis
•	Code development principles and contribution of new models
•	Overview of LWR, TRISO, and Metallic fuel capabilities
•	Meshing fuel geometries
•	Simulation output analysis and visualization
•	Interactive practice examples and in-person support
 
The deadline for registration to the BISON workshop is December 1st, 2023. We encourage early registration as space is limited.
 
 
The sign-up link is at this address for both training. If you want to attend both, you must register twice, once for each training date. 
https://ncrcaims.inl.gov/Identity/Account/TrainingRegistration

**Comment by CM518** [comment]
Hi, do you know when we will get confirmation that we have been accepted onto the workshops, I know the places are limited?

**Reply by GiudGiud** [reply]
Hello

We are yet to process but I m pretty sure we will accept everyone

Guillaume
